packages/native-federation-typescript/src/lib/typeScriptCompiler.ts

```diff
@@ -74,16 +77,23 @@ const createHost = (
     );
 
     for (const sourceFile of sourceFiles || []) {
-      const sourceEntry = mapExposeToEntry[sourceFile.fileName];
+      const sourceEntry = mapExposeToEntry[normalize(sourceFile.fileName)];
       if (sourceEntry) {
         const mfeTypeEntry = join(
           mfTypePath,
           `${sourceEntry}${DEFINITION_FILE_EXTENSION}`,
         );
         const mfeTypeEntryDirectory = dirname(mfeTypeEntry);
-        const relativePathToOutput = relative(mfeTypeEntryDirectory, filepath)
+        let relativePathToOutput = relative(mfeTypeEntryDirectory, filepath)
```

Comment:
same as the previous comment

Reply:
Yeah, but I specially did it as well as in @module-federation/typescript...
Okay, I will fix it